extensions/amp-a4a/0.1/amp-a4a.js

```diff
@@ -738,6 +739,9 @@ export class AmpA4A extends AMP.BaseElement {
           // Preload any fonts.
           (creativeMetaDataDef.customStylesheets || []).forEach(font =>
               this.preconnect.preload(font.href));
+          // Preload any AMP images.
+          (creativeMetaDataDef.ampImages || []).forEach(image =>
+              isSecureUrl(image) && this.preconnect.preload(image));
```

Comment:
LGTM. @dvoytenko to confirm, we are ok to preload an array of images?

Reply:
Yeah. We discussed. Should be ok as long as this list is not super long.